openwisp_firmware_upgrader/static/firmware-upgrader/css/batch-upgrade-operation.css

```diff
@@ -0,0 +1,121 @@
+/* Batch Upgrade Operation CSS */
```

Comment:
Please remove this comment because the name of the file already says this, redundant comments are not useufl.